src/components/Create/components/RewardsList/AddEditRewardModal.tsx

```diff
@@ -236,19 +282,108 @@ export const AddEditRewardModal = ({
             )}
           </Space>
         </Form.Item>
-        <Form.Item
-          name="externalUrl"
-          label={t`External link`}
-          rules={[
-            inputIsValidUrlRule({
-              label: t`External link`,
-              prefix: 'https://',
-              validateTrigger: 'onCreate',
-            }),
-          ]}
+        <CreateCollapse
+          activeKey={advancedOptionsOpen ? ['0'] : []}
+          onChange={onCollapseChanged}
         >
-          <PrefixedInput prefix="https://" />
-        </Form.Item>
+          <CreateCollapse.Panel
+            header={<OptionalHeader header={t`Advanced options`} />}
+            key={0}
+            hideDivider
+          >
+            <Form.Item
+              extra={
+                <span className="text-sm">
+                  <Trans>Set aside a percentage of freshly minted NFTs.</Trans>
+                </span>
+              }
+            >
+              <JuiceSwitch
+                value={isReservingNfts}
+                onChange={setIsReservingNfts}
+                label={
+                  <span className="flex items-center gap-2">
+                    <Trans>Reserve NFTs</Trans>
```

Comment:
Should this be 'Reserved NFTs' do you think? I'm thinking mainly to make it clear it's a similar concept to 'Reserved tokens'